plugin/src/main/java/org/opensearch/ml/utils/MLExceptionUtils.java

```diff
@@ -20,6 +20,8 @@ public class MLExceptionUtils {
     public static final String NOT_SERIALIZABLE_EXCEPTION_WRAPPER = "NotSerializableExceptionWrapper: ";
     public static final String REMOTE_INFERENCE_DISABLED_ERR_MSG =
         "Remote Inference is currently disabled. To enable it, update the setting \"plugins.ml_commons.remote_inference_enabled\" to true.";
+    public static final String LOCAL_MODEL_DISABLED_ERR_MSG =
+        "Local Model Inference is currently disabled. To enable it, update the setting \"plugins.ml_commons.local_model.enabled\" to true.";
```

Comment:
Should we also change this part? As we are not just disable inference

Reply:
fixed it